iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/converter/ProjectToOrganizationConverter.java

```diff
@@ -0,0 +1,70 @@
+package eu.dnetlib.iis.wf.importer.converter;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NavigableMap;
+
+import org.apache.hadoop.hbase.client.Result;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+
+import eu.dnetlib.data.proto.OafProtos.Oaf;
+import eu.dnetlib.data.proto.OafProtos.OafRel;
+import eu.dnetlib.iis.importer.schemas.ProjectToOrganization;
+import eu.dnetlib.iis.wf.importer.OafHelper;
+import eu.dnetlib.iis.wf.importer.input.approver.ResultApprover;
+
+/**
+ * {@link ProjectToOrganization} converter.
+ * @author mhorst
+ *
+ */
+public class ProjectToOrganizationConverter extends AbstractAvroConverter<ProjectToOrganization[]> {
```

Comment:
Right, this is the con of the `*To*` naming convention. Let's solve it as suggested in comment above.